Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -560,6 +560,29 @@ public void TestPutGetGcsDownscopedCredential(
             }
         }
 
+        [Test]
+        public void TestPutFileWithSpaceAndSingleQuote()
```

Comment:
what about Get operation? Maybe you could create an integration test for getting files with spaces in their names as well?

Reply:
Modified test to include GET on a file path with space